packages/terra-paginator/tests/jest/Paginator.test.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Paginator from '../../src/Paginator';
+
+describe('Paginator', () => {
+  const defaultRender = <Paginator onPageChange={e => typeof e} selectedPage={1} totalCount={2234} itemCountPerPage={20} />;
+  const noPagesRender = <Paginator onPageChange={e => typeof e} />;
+
+  // Snapshot Tests
+  it('should render a default paginator', () => {
+    const wrapper = shallow(defaultRender);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should render a paginator without page links', () => {
+    const wrapper = shallow(noPagesRender);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  // Structure Tests
+  it('should have the class paginator', () => {
+    const wrapper = shallow(defaultRender);
```

Comment:
Do we need this test? I believe your snapshots that you generated cover this case

Reply:
https://github.com/cerner/terra-core/pull/1445/commits/c619c108cd3f92c3084a93d5dc3d99e53bd717ba